### DISCUSSION TITLE
ProxyJump for Server setting

### DISCUSSION THREAD

**Discussion by IlyaFed** [isontheline/pro.webssh.net#1057] [Ideas]
Hello, 

I’m very glad that I bought your application, for me it is the most convenient on the iPad!

But there is one very useful thing in SSH, which I often use, since most of the servers are behind a proxy server:
ProxyJump (ssh -J )
https://man.openbsd.org/ssh_config#ProxyJump

Maybe it can be added to the server config so that you can select the server through which to log in? This would make it much easier to use!

**Comment by isontheline** [comment]
Hello @IlyaFed 👋

Many thanks for your feedback!

While the jump feature is being implemented #577, you can use [Dynamic Port Forwarding](https://webssh.net/documentation/help/networking/dynamic-port-forwarding/) to do the same thing you need aka jump over a bastion.

Could you give it a try?

**Comment by IlyaFed** [comment]
Thank you very much, the Dynamic Port Forwarding really works and I will use it! But I will also still wait for the implementation of ProxyJump, because it will reduce the connection steps and allow me to synchronize with my ~/.ssh/config on the Mac.

**Reply by isontheline** [reply]
No problem I will work on ProxyJump
Many thanks!
